### DISCUSSION TITLE
Welcome to disk.frame Discussions!

### DISCUSSION THREAD

**Discussion by xiaodaigh** [xiaodaigh/disk.frame#330] [General]
## 👋 Welcome!

From now on we should use Discussion to do the following and leave the issues tab for real issues. Thank you!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clem12240** [comment]
Hello ZJ, 

I'm currently working on a secured server where I can only get a limited amount of RAM. 
I'm very new to R. I have been using the package felm in order fit linear, twoway fixed effects models. 
It is very convenient as it identifies the "connected components"of the dataset. 

However, in my latest application, I struggle to fit the model as the data frame is too large. 

I was wondering whether the package disk.frame would be useful in that setting... 

Kind regards, 

Clément

**Reply by clem12240** [reply]
Maybe I was supposed to write some other place, I apologize if this spot is not appropriate...

**Reply by xiaodaigh** [reply]
> Maybe I was supposed to write some other place, I apologize if this spot is not appropriate...

It's fine. You could've started a new discussion too I think. 

Does {felm} have progressive fit abilities? Or partial fit abilities? If it has then disk.frame might be able to help.

Say if I have dataset `a1` and `a2` which together is my training data. Can I train `{felm}` to fit on dataset `a1` and get some coefficients, then add `a2` to the model and update the coefficients? If {felm} can operate like that then disk.frame can help.

**Comment by clem12240** [comment]
Thank you for your answer. I'll look into whether {felm} would allow me to
update sequentially my estimates.



On Thu, Mar 25, 2021 at 1:05 PM evalparse ***@***.***> wrote:

> Maybe I was supposed to write some other place, I apologize if this spot
> is not appropriate...
>
> It's fine. You could've started a new discussion too I think.
>
> Does {felm} have progressive fit abilities? Or partial fit abilities? If
> it has then disk.frame might be able to help.
>
> Say if I have dataset a1 and a2 which together is my training data. Can I
> train {felm} to fit on dataset a1 and get some coefficients, then add a2
> to the model and update the coefficients? If {felm} can operate like that
> then disk.frame can help.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/xiaodaigh/disk.frame/discussions/330#discussioncomment-529290>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACPE7Z3IOF35B5K46O7ZRWLTFMRJ7ANCNFSM4ZSAYQPA>
> .
>


-- 
Clément Malgouyres
Économiste, Institut des politiques publiques (IPP) www.ipp.eu,
Chercheur affilié à Paris School of Economics (PSE)
Site web: https://sites.google.com/site/clementmalgouyres
